prediction_market_agent/agents/crewai_subsequential_agent/deploy.py

```diff
@@ -0,0 +1,75 @@
+from decimal import Decimal
+import random
+from prediction_market_agent_tooling.deploy.agent import DeployableAgent
+from prediction_market_agent_tooling.markets.agent_market import AgentMarket
+from prediction_market_agent_tooling.markets.data_models import BetAmount, Currency
+from prediction_market_agent_tooling.markets.markets import MarketType
+
+from prediction_market_agent.agents.crewai_subsequential_agent.crewai_agent_subquestions import CrewAIAgentSubquestions
+from prediction_market_agent.agents.known_outcome_agent.known_outcome_agent import (
+    Result,
+)
+from prediction_market_agent.agents.utils import market_is_saturated
+
+
+class DeployableThinkThoroughlyAgent(DeployableAgent):
+    # For cheaper credits at this experimental stage
+    model = "gpt-3.5-turbo"
+
+    def load(self) -> None:
+        self.markets_with_known_outcomes: dict[str, Result] = {}
```

Comment:
Is this new agent also meant to answer only knownoutcomes markets?

Reply:
Nope, removed this method

Reply:
Seems like it's still there

Reply:
I hadn't committed, now it was removed -> https://github.com/gnosis/prediction-market-agent/commit/65131384e0fdd8ed397955dfe611fe4890e61b18#diff-801f2ee8ccabc0596d3c2fcd7d2198c8b9b3a6af0f093a5d8c04295d9ae6e4c0L22-L25